index.js

```diff
@@ -29,6 +29,9 @@ app.use("/fonts", express.static("./assets/fonts"))
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(sharify)
+// Inject METAPHYSICS_ENDPOINT to sharify data
+// Todo/Enhancement: inject any 'safe ENV vars?
+sharify.data.METAPHYSICS_ENDPOINT = process.env.METAPHYSICS_ENDPOINT
```

Comment:
What I mean, is that for development, I added this line in this file...which will make things work in any app/component that uses it.

However, the way we export apps/components _won't_ include this, and so those exported bits of code will be expecting/assuming some sharify data. 

I suppose that's fine, for any places that use this to make sure it's available, Force does already do this. I'm not 100% sure what the implications might be (or if there are any) for a component that we want to use on mobile, that isn't using sharify at all? But then would `res.locals.sharify.data` _should_ still work?

I think either way we can cross that bridge when we get there, since this should all be fine for Force, but something feels weird here to me, a little bit.